src/WebSdk/Worker/Targets/Microsoft.NET.Sdk.Worker.targets

```diff
@@ -25,4 +26,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ProjectCapability Include="WebNestingDefaults" />
   </ItemGroup>
 
+
+  <ItemGroup Condition=" '$(DisableImplicitNamespaceImports_Worker)' != 'true'
+      And '$(TargetFrameworkIdentifier)' == '.NETCoreApp'
+      And $([MSBuild]::VersionGreaterThanOrEquals('$(_TargetFrameworkVersionWithoutV)', '6.0'))">
```

Comment:
I don't believe this condition is required anywhere besides Microsoft.NET.GenerateImplicitNamespaceImports.targets (and probably only needed on `GenerateImplicitNamespaceImports` target). 
Few reasons:
- nothing will be emitted unless the target is run, and
- this makes it extremely brittle - if the condition changes it will need to be applied universally and in multiple places, including SDKs that aren't part of this repo (e.g. Windows Desktop, which resides in dotnet/wpf and dotnet/windowsdesktop, but has bits ingested from dotnet/winforms).

Reply:
E.g. this is how I expect the Windows Forms global usings props to look:
```xml
<Project>

  <ItemGroup Condition=" '$(UseWindowsForms)' == 'true' ">
    <!--
      SDK defines the following global usings:
      * System
      * System.Collections.Generic
      * System.Linq
      * System.Threading.Tasks
    -->
    <Import Include="System.Drawing" />
    <Import Include="System.Windows.Forms" />
  </ItemGroup>

</Project>
```

Referenced through the Microsoft.NET.Sdk.WindowsDesktop.props this leads to the following:
![image](https://user-images.githubusercontent.com/4403806/124079995-2c18d180-da8d-11eb-82aa-e7d9184dd1f9.png)

Reply:
We actually want each SDK to specify their own list of namespaces that are TFM driven to ensure compatibility. Any changes to the list of namespaces are breaking changes so we want to guard them with TFM checks in each SDK.

I think it's valuable to add a TFM check for > net6.0 for the entire feature though. 

FYI the CS0185 warning is going to be removed. Let me find the issue and link it here.

Reply:
> We actually want each SDK to specify their own list of namespaces that are TFM driven to ensure compatibility. Any changes to the list of namespaces are breaking changes so we want to guard them with TFM checks in each SDK.

Alright. Though IMO it should be additive to the base list.

Reply:
I think each SDK that adds `Import` items should condition it on the target framework identifier and version.  That allows us to change the default imports over time without breaking existing projects (until they are retargeted).

I think these `Import` items should be moved to `Microsoft.NET.Sdk.Worker.props`.